src/main/kotlin/com/hypto/iam/server/apis/TokenApi.kt

```diff
@@ -42,7 +42,7 @@ fun Route.tokenApi() {
         }
     }
 
-    authenticate("unique-basic-auth", "basic-auth", "bearer-auth") {
+    authenticate("unique-basic-auth", "bearer-auth") {
```

Comment:
Removed `basic-auth` as it requires organizationId from parameters and `/token` does not have it